agent/README.md

```diff
@@ -73,12 +73,16 @@ __Please check the produced log file for errors and warnings before using the ag
 
 The log file is written to the agent's directory in the subdirectory `logs` by default. If there is no log file at that location, it means the agent didn't even start and you have not configured it correctly.
 
+#### Testwise coverage
+
+If you want to collect testwise coverage, please have a look below in the [Testwise mode section](####Testwise-coverage-modes).
+
 #### Path format
 
 All paths supplied to the agent can be absolute or relative to the working directory. Furthermore paths may contain ant
 patterns with `*`, `**` and `?`.
 
-### Options for normal mode
+### Options for normal mode (do not apply to testwise mode)
```

Comment:
I would use the same formulation for normal mode and testwise mode (Options for vs. Testwise mode (options))